src/test/shared/clients/codeCatalystClient.test.ts

```diff
@@ -0,0 +1,53 @@
+/*!
```

Comment:
Tests for the CC client already live in `./src/test/caws/codecatalystClient.test.ts`
(which should probably be renamed to `./src/test/codecatalyst/codecatalystClient.test.ts` )

Reply:
Oops I forgot this depends on https://github.com/aws/aws-toolkit-vscode/pull/3252

Reply:
Should have updated this PR description to address the confusion. This is fixed in #3252 as you indicated.